tests/src/python/test_offline_editing_wfs.py

```diff
@@ -52,6 +52,7 @@ class TestWFST(unittest.TestCase, OfflineTestBase):
     @classmethod
     def setUpClass(cls):
         """Run before all tests"""
+        super(TestWFST, cls).setUpClass()
```

Comment:
Yep - this had me confused as well. But it turns out this is the only way to reliably get the unittest method called!

The good thing is that this is kind of self-testing, because if the unittest method isn't called then the test will raise an exception when the tear down method is called.